client/src/store/modules/profile.js

```diff
@@ -32,17 +32,47 @@ export default {
       axios
         .get('/api/profiles/get-followers/' + id)
         .then(response => {
-          commit('updateFollowers', response.data)
+          if (response.data.followers === "This profile doesnt have followers") {
```

Comment:
What will happen if a message in response will change? Would it be better to check if response.data.followers is a non-empty array? if empty updateFollowers([]) else updateFollowers(response.data.followers)